tests/npm-app/app.json

```diff
@@ -178,7 +178,6 @@
                 "application/json": {
                   "schema": {
                     "items": {
-                      "items": {},
```

Comment:
Why is this necessary?

Reply:
The validator we're using doesn't like this empty object - it doesn't think this matches the schema for a JSON schema object. So we say the same thing ("the items have no further type restriction") by omitting the field entirely.